.github/workflows/ci-test-and-push-docker.yml

```diff
@@ -159,4 +173,5 @@ jobs:
 
       - name: Inspect image
         run: |
-          docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:${{ steps.meta.outputs.version }}
```

Comment:
The Image-Tag comes from the metadata-action (142-146).
steps.meta.outputs.version results in a cleaner naming convention than you approach. I´d suggest to just leave this as is.
Read more at:
https://github.com/docker/metadata-action